packages/core/src/lib/passcode.ts

```diff
@@ -0,0 +1,30 @@
+import { PasscodeType } from '@logto/schemas';
+import { customAlphabet, nanoid } from 'nanoid';
+
+import {
+  deletePasscodesByIds,
+  findUnconsumedPasscodesBySessionIdAndType,
+  insertPasscode,
+} from '@/queries/passcode';
+
+const randomCode = customAlphabet('1234567890', 6);
```

Comment:
should we make the length of the randomCode / passcode configurable? or provide a 4-digit random code generator as well?

Reply:
Change into a constant.